lib/theme.json

```diff
@@ -357,6 +357,30 @@
 					}
 				}
 			},
+			"core/form-input": {
+				"elements": {
+					"textInput": {
+						"spacing": {
+							"margin": {
+								"bottom": "0.5em"
+							},
+							"padding": {
+								"top": "0",
+								"right": "0.5em",
+								"bottom": "0",
+								"left": "0.5em"
+							}
+						},
+						"typography": {
+							"lineHeight": "2",
```

Comment:
I feel like `2` is a bit too big, what do you think?

<img width="692" height="412" alt="image" src="https://github.com/user-attachments/assets/b2c16b96-4eb4-4b57-8013-2b875e7a4384" />

Reply:
Yep that is a bit big. 1.6 is usually a good middle.

Reply:
Fixed this on [packages/blocks/src/api/constants.js](https://github.com/WordPress/gutenberg/pull/70378/files#diff-06eb310f801cd86df6d25f97f3aae56297e59277c8bef216729456f69ede32fc)